meta-genivi-dev/recipes-extended/automotive-message-broker/automotive-message-broker_git.bb

```diff
@@ -17,6 +17,7 @@ SRCREV = "c216955d16ca275159891cad296217094d972390"
 
 SRC_URI += "file://amb_allow_sessionbus.patch \
             file://ambd.service \
+            file://ambd_fsa.service \
```

Comment:
In fact, it is a amb instance with a particular config file that is used to simulate data. It is used for FSA but it is an amb releated.
I see like dbus: 
    - There is a service for dbus session
    - There is a service for dbus server
but at the end the process is still dbus-dameon with parameters specific.